subsys/bluetooth/controller/ll_sw/nordic/hal/nrf5/mayfly.c

```diff
@@ -70,14 +67,7 @@ u32_t mayfly_is_enabled(u8_t caller_id, u8_t callee_id)
 u32_t mayfly_prio_is_equal(u8_t caller_id, u8_t callee_id)
 {
 	return (caller_id == callee_id) ||
-#if defined(CONFIG_BT_LL_SW_LEGACY)
-#if (CONFIG_BT_CTLR_WORKER_PRIO == CONFIG_BT_CTLR_JOB_PRIO)
-	       ((caller_id == MAYFLY_CALL_ID_WORKER) &&
-		(callee_id == MAYFLY_CALL_ID_JOB)) ||
-	       ((caller_id == MAYFLY_CALL_ID_JOB) &&
-		(callee_id == MAYFLY_CALL_ID_WORKER)) ||
-#endif
-#elif defined(CONFIG_BT_LL_SW_SPLIT)
+#if defined(CONFIG_BT_LL_SW_SPLIT)
```

Comment:
Is the `#if` needed anymore now?

Reply:
No, it's not. But I intentionally left it there as a reminder of potential future forks of the LL.

Reply:
Ok. I was assuming that we wouldn't do that.